src/message-page.scss

```diff
@@ -0,0 +1,86 @@
+@import "./new-settings";
+@import "./mixins";
+
+$fd-message-page: 0 !default;
+
+$block: #{$fd-namespace}-message-page;
+
+.#{$block} {
+  @include fd-reset();
+  @include fd-flex-center();
+
+  width: 100%;
+  height: 100%;
+  padding: 1rem;
+  background: var(--sapBackgroundColor);
+
+  &__container,
+  &__content,
+  &__title,
+  &__subtitle,
+  &__actions,
+  &__more {
+    @include fd-reset();
+
+    max-width: 30rem;
```

Comment:
Setting up max-width on the container wouldn't be enough?

Reply:
nope :) it's for IE11